test/CardsTests.cpp

```diff
@@ -0,0 +1,41 @@
+#include <gtest/gtest.h>
+#include "Cards.hpp"
+#include <stdexcept>
+
+struct CardsTests : public ::testing::Test
+{
+    json dataJson = R"({
```

Comment:
It would be better to keep this info in a separate json file and read it in this test file.

Reply:
I need only 1 card for test this class, so I make it in class CardsTests - than test will be independent of the size of input json file ;-).